CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+#👉 How to contribute to Giskard?
+
+Everyone is encouraged to contribute, and we appreciate each and every one of them. Helping the community is thus not limited to writing code. The community will greatly benefit from your efforts to provide clarification, assist others, and enhance the documentation. 📗
+
+Additionally, it is helpful if you help us spread the word by mentioning the library in blog articles about the amazing projects it enabled, tweeting about it on occasion, or just staring the repository! ⭐️
+
+If you choose to contribute, please be mindful to respect our [code of conduct](https://github.com/Giskard-AI/giskard/blob/main/CODE_OF_CONDUCT.md).
+
+## The different ways you can contribute to Giskard!
+
+There are 4 ways you can contribute to Giskard:
```

Comment:
I would order by importance
1. Submitting issues
2. Contributing to examamples
3. Fixing issues
4. Implementing tests